src/renderer/components/sidebar-item.js

```diff
@@ -136,6 +136,7 @@ class SidebarItem extends PureComponent {
         ? [
             {
               label: label('lock'),
+              accelerator: `CmdOrCtrl+l`,
```

Comment:
make this conditional: `this.props.active ? ... : null`, because if the archive is not active, the shortcut wont work.